bower.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "google-feeds",
+  "version": "0.8.0",
+  "homepage": "https://github.com/GoogleWebComponents/google-feeds",
+  "authors": [
+    "Addy Osmani <addyosmani@gmail.com>"
+  ],
+  "description": "Google Feeds element for Polymer",
+  "main": "google-feeds.html",
+  "keywords": [
+    "web-components",
+    "polymer",
+    "feeds",
+    "feeds",
+    "api"
+  ],
+  "license": "Apache-2",
+  "dependencies": {
+    "polymer": "Polymer/polymer#v0.8.0-rc.7",
```

Comment:
We're currently stuck on pre-0.9rc until the latest round of gesture bugs get resolved. It's currently busting Starter Kit and paper-drawer-panel :/